x/programs/examples/token.go

```diff
@@ -110,22 +110,15 @@ func (t *Token) Run(ctx context.Context) error {
 	)
 
 	// initialize program
-	resp, err := rt.Call(ctx, "init", programContext)
+	_, err = rt.Call(ctx, "init", programContext)
 	if err != nil {
 		return fmt.Errorf("failed to initialize program: %w", err)
 	}
 
-	t.log.Debug("init response",
-		zap.Int64("init", resp[0]),
-	)
-
-	result, err := rt.Call(ctx, "get_total_supply", programContext)
```

Comment:
It's a pain that these results were not asserted at all. As it's not used anymore since the temporary removal of the big integration tests, it's probably fine to wait to get them back. In the meantime, you could use 
```go
	t.log.Debug("init response",
		zap.ByteString("init", resp),
	)
```

<img width="839" alt="Screenshot 2024-05-14 at 01 18 10" src="https://github.com/ava-labs/hypersdk/assets/51274081/7d92d5d5-9055-437c-93f7-79c4a8c88abe">

Reply:
Yeah, we can add them back later